app/views/content/_id_requirements.html.erb

```diff
@@ -1 +1 @@
-<%= markdown(prison_specific_id_requirements(visit.prisoner.prison_name)) %>
+<%= markdown(prison_specific_id_requirements(visit.prison)) %>
```

Comment:
If `prison` is available via the helper now can we just do:

```ruby
<%= markdown(prison_specific_id_requirements(prison)) %>
```

Same would apply for the `text.er` of this template.